platformcomponents/desktop/navigation-bar.json

```diff
@@ -0,0 +1,10 @@
+{
+  "navigation-bar": {
+    "comment": "This is applied to the first column navigation bar.",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/branch/WG6O99kWm33upbqMdnnIQA/Components---Windows%2BWeb?node-id=29743%3A57187",
```

Comment:
this link takes me to compose area component; it should be: https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=5250%3A957